test/arquillian-extension/src/main/java/io/narayana/lra/arquillian/spi/NarayanaLRARecovery.java

```diff
@@ -60,4 +76,13 @@ private boolean recoverLRAs(URI lraId) {
             return !json.contains(lraId.toASCIIString());
         }
     }
+
+    private static Integer initWaitForCallbackTimeout() {
+        try {
+            return ConfigProvider.getConfig().getValue(LRAConstants.WAIT_CALLBACK_TIMEOUT, Integer.class);
```

Comment:
There are better APIs then catching an exception. Or do you expect that config doesn't have to be present? We should refactor all config properties into the same access.

Reply:
When you are saying the "same access" are you referring to the same style? For example, to be consistent with https://github.com/jbosstm/lra/blob/main/jaxrs/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java#L460? Or are you saying to accessing all the config in one go?

Reply:
Thanks for you review @xstefank , looking at the Config.getValue()  method you can see that it returns "java.util.NoSuchElementException  if the property is not defined or is defined as an empty string or the converter returns"
However I will update the try catch block.

Reply:
Yes, definetely use optional, and I also agree that c9ngig should be parsed at one place. But this is a test, so it's ok.